mlflow/deployments/databricks/__init__.py

```diff
@@ -0,0 +1,44 @@
+from mlflow.deployments import BaseDeploymentClient
+
+
+class DatabricksDeploymentClient(BaseDeploymentClient):
+    def create_deployment(self, name, model_uri, flavor=None, config=None, endpoint=None):
```

Comment:
Is the model_uri going to be required if we're creating a gateway route, or is gateway route creation purely going to be handled with `create_endpoint()` ?

Reply:
do we need the flavor designator here? If we're using model_uri, can it read the configured flavor information from the MLmodel file?

Reply:
> is gateway route creation purely going to be handled with create_endpoint() ?

yes

Reply:
perfect!